src/utils/url.ts

```diff
@@ -85,7 +85,7 @@ export function addSubdomain(url: string, subdomain: string): string {
  * @param query - The query parameters.
  * @returns - The final URL.
  */
-export function addUrlQuery(url: string, query: Record<string, unknown>): string {
+export function addUrlQuery(url: string, query: { [key: string]: string | undefined }): string {
```

Comment:
```suggestion
export function addUrlQuery(url: string, query: { [key: string]: string | number | boolean }): string {
```

is undefined possible here? in any case we should keep supporting all basic types - namely numbers and booleans included